tests/other/zaccommodation.test.js

```diff
@@ -0,0 +1,124 @@
+import chai, { expect } from 'chai';
```

Comment:
can you rename this test file? `zaccommodation.test.js` is not really descriptive, please find a descriptive name.

Reply:
I did this for the sake of other previous tests which the accommodations tests depends on